packages/web-components/src/components/feature-card-block-large/feature-card-block-large.scss

```
@@ -0,0 +1,14 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+@import '../../globals/scss/vars';
+@import '@carbon/ibmdotcom-styles/scss/components/card/index';
+@import '@carbon/ibmdotcom-styles/scss/components/feature-card-block-large/feature-card-block-large';
+:host(#{$dds-prefix}-feature-card-block-large) {
```

Coding style nit, to align to other files in the codebase:

```suggestion

:host(#{$dds-prefix}-feature-card-block-large) {
```